bin/cron/deliver_poste_messages

```diff
@@ -72,16 +72,27 @@ def main
         )
         deliverer.reset_connection
         sent_at = 0
-      rescue AbortEmailError => exception
+      rescue AbortEmailError, Psych::SyntaxError => exception
         Honeybadger.notify(
           exception,
-          error_message: "Abandoning delivery of #{delivery[:id]} because '#{exception.message.to_s.strip}'"
+          error_message: "Abandoning delivery of #{delivery[:id]} because '#{exception.message.to_s.strip}'",
+          context: {
+            delivery: {
+              id: delivery[:id],
+            }
+          }
         )
         sent_at = 0
       rescue => exception
         Honeybadger.notify(
           exception,
-          error_message: "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{exception.message.to_s.strip}'"
+          error_message: "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{exception.message.to_s.strip}'",
```

Comment:
Should we consider removing PII from these logs? Is the email address essential for debugging?